ReSharper.FSharp/src/FSharp/FSharp.Common/src/ProjectModel/FSharpLanguageVersion.fs

```diff
@@ -31,7 +31,10 @@ type FSharpLanguageLevel =
     /// Nested record field copy and update/shorthand lambda
     | FSharp80 = 80
 
-    | Latest = 80
+    /// Nullness
+    | FSharp90 = 90
+
+    | Latest = 80 //TODO
```

Comment:
If changed to 90, the tests will fail with

![image](https://github.com/user-attachments/assets/d81d4b7b-4b2e-4c40-8082-83d94a16730e)

Reply:
Thanks! We'll need to update FCS to fix it.